src/Storage/custom/latest-2019-04-30/Set-AzRmStorageContainerImmutabilityPolicy.ps1

```diff
@@ -0,0 +1,122 @@
+<#
+.Synopsis
+Creates or updates an unlocked immutability policy. ETag in If-Match is honored if given but not required for this operation.
+.Description
+Creates or updates an unlocked immutability policy. ETag in If-Match is honored if given but not required for this operation.
+.Example
+To view examples, please use the -Online parameter with Get-Help or navigate to: https://docs.microsoft.com/en-us/powershell/module/az.storage/set-azrmstoragecontainerimmutabilitypolicy
+.Inputs
+Microsoft.Azure.PowerShell.Cmdlets.Storage.Models.Api20180201.IImmutabilityPolicy
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.Storage.Models.Api20180201.IImmutabilityPolicy
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.storage/set-azrmstoragecontainerimmutabilitypolicy
+#>
+function Set-AzRmStorageContainerImmutabilityPolicy {
```

Comment:
This should have a parameter set that accepts the storage account object

Reply:
As discussed we agreed on not having a parameter set for just storage accounts here.